payment_cod/models/payment_transaction.py

```diff
@@ -25,8 +25,16 @@ def _process_feedback_data(self, data):
 
     def _finalize_post_processing(self):
         for tx in self:
-            if tx.acquirer_id.is_cod:
-                # We do not process payment and reconciliation by cron for COD cases.
+            if not tx.acquirer_id.is_cod:
+                super(PaymentTransaction, tx)._finalize_post_processing()
                 continue
-            super(PaymentTransaction, tx)._finalize_post_processing()
+            # Below steps replicate the relevant parts from _reconcile_after_done() in
+            # sale module.
+            # The order will not be confirmed without shit part
```

Comment:
```suggestion
            # The COD order will not be confirmed without these.
```
No swearing please. :rofl: